ocs_ci/templates/openshift-infra/logging-deployment/clusterlogging_operator/instance.yaml

```diff
@@ -15,8 +15,8 @@ spec:
           cpu: "600m"
           memory: "4Gi"
       storage:
-        storageClassName: "gp2"
-        size: "200G"
+        storageClassName: "rook-ceph-ceph-rbd"
+        size: "100G"
```

Comment:
isn't this contradicting with `inst_data['spec']['logStore']['elasticsearch']['storage']['size'] = "200Gi"`

Reply:
Done